### ISSUE TITLE
Legacy pages will not work along side of Goomento pages, its either or 

### ISSUE THREAD

**Issue by fwddev-pgh** [Goomento/PageBuilder#30]
Hello, I have an issue where my legacy M2 pages will not work when Goomento is activated. I can create a NEW page using the Goomento landing page builder using the same URL, and the page will work, but all existing CMS pages will 404. 

Can you make it so you can keep your legacy pages and use the Landing page builder for NEW pages? I plan on moving all over to Goomento as we change old pages. 

I have tried to open existing pages and convert them to Goomento using the "Migrate Now" button without success. 

In summary, legacy pages not working all go to 404 not found. 

**Works** -> Goomento -> Page Builder | Pages - Landing Pages 
**Does not work**  -> Content -> Elements | Pages

If you need anything else to assist with fixing the issue, please let me know, also thanks for a great product.

**Comment by haihv433** [comment]
Hi @fwddev-pgh 
Thanks for caring, 

Can you help us to provide 
- Magento version 
- Goomento version 
- PHP version
If possible can you send us an video to demonstrate or Magento instance/SSH to deeper check, 

Thanks

**Comment by fwddev-pgh** [comment]
2.4.6 / PHP 8.1.13
Goomento 4.3

I will send you a video and pictures to your email

**Comment by haihv433** [comment]
Hi @fwddev-pgh 

I answered your email, 

Feel free to re-open this if issue still persists,

Thanks
